app/src/main/java/com/android/streetworkapp/model/progression/Progression.kt

```diff
@@ -69,6 +69,11 @@ enum class MedalsAchievement(val achievement: Achievement) {
           "Last medal"))
 }
 
+enum class ExerciseAchievement(val achievement: Achievement) {
```

Comment:
I see that it hasn't been done in the other `enum class` statements in this file, but it would be a good idea to put a short description of what this class represents.

Reply:
I think you are right. I added a description for every enum in this class.